config/locales/en.yml

```diff
@@ -1,5 +1,6 @@
 en:
   errors:
+    incorrect_password: is incorrect
```

Comment:
No, because the attribute that it is tied to gets automatically added to the error message. If we made this "Current password is incorrect", then the error message would be "Current password Current password is incorrect".